bot/bot.py

```diff
@@ -55,6 +69,14 @@ async def close(self) -> None:
         if self.redis_session:
             await self.redis_session.close()
 
+    def load_extension(self, name: str, *, package: Any = None) -> None:
```

Comment:
This should be `Optional[str]`.
```suggestion
    def load_extension(self, name: str, *, package: Optional[str] = None) -> None:
```